pkg/operator/sync_openshiftcontrollermanager_v311_00.go

```diff
@@ -153,6 +160,11 @@ func syncOpenShiftControllerManager_v311_00_to_latest(c OpenShiftControllerManag
 	operatorConfig.Status.ObservedGeneration = operatorConfig.ObjectMeta.Generation
 	resourcemerge.SetDaemonSetGeneration(&operatorConfig.Status.Generations, actualDaemonSet)
 
+	return processReturn(c, errors, originalOperatorConfig, operatorConfig)
+
+}
+
+func processReturn(c OpenShiftControllerManagerOperator, errors []error, originalOperatorConfig, operatorConfig *operatorapiv1.OpenShiftControllerManager) (bool, error) {
```

Comment:
nit - perhaps rename to `syncReturn` or `finishSync`

Reply:
switched to `syncReturn` @adambkaplan